### ISSUE TITLE
Chrome Extension "May Have Been Corrupted" Error

### ISSUE THREAD

**Issue by asteino** [EFForg/https-everywhere#8216]
The Chrome Extension keeps showing an "This extension may have been corrupted" error message. I've tried to repair the extension and I've also tried deleting + re-installing with the same result. 
![maybecorrupted](https://cloud.githubusercontent.com/assets/1445222/22039695/ea24b826-dcc5-11e6-8b4e-8a141eca8181.PNG)

**Comment by jecal22** [comment]
I have noticed the same issue.  I did a complete removal of Chrome and all data this morning and after re-installing and signing back into Chrome I noticed that HTTPS Everywhere was no longer running.  I see the same message when I visit the Extensions page. 

I also have tried the Repair option with no luck as well as removing the extension and re-adding from the Chrome store.  I've signed into a second Chrome profile and the same issue occurs.
